R/derive_extreme_records.R

```diff
@@ -1,46 +1,36 @@
 #' Add the First or Last Observation for Each By Group as New Records
 #'
 #' Add the first or last observation for each by group as new observations. The
-#' new observations can be selected from the input dataset or an additional
-#' dataset. This function can be used for adding the maximum or minimum value
-#' as a separate visit. All variables of the selected observation are kept. This
-#' distinguishes `derive_extreme_records()` from `derive_summary_records()`,
+#' new observations can be selected from the source dataset. This function can
+#' be used for adding the maximum or minimum value as a separate visit.
+#' All variables of the selected observation are kept. This distinguishes
+#' `derive_extreme_records()` from `derive_summary_records()`,
 #' where only the by variables are populated for the new records.
 #'
 #' @param dataset `r roxygen_param_dataset()`
 #'
-#'   If `dataset_add` is not specified, the new records are selected from the
-#'   input dataset. In this case the variables specified by `by_vars` and
-#'   `order` are expected.
-#'
 #' @param dataset_ref Reference dataset
 #'
 #'   The variables specified for `by_vars` are expected. For each
 #'   observation of the specified dataset a new observation is added to the
 #'   input dataset.
 #'
-#' @param dataset_add Additional dataset
-#'
-#'   Observations from the specified dataset are added as new records to the
-#'   input dataset (`dataset`).
+#' @param dataset_add Source dataset
```

Comment:
I would use the term "additional dataset" to be consistent with the other functions.

Reply:
Updated, thanks!